### DISCUSSION TITLE
Support for Debugger

### DISCUSSION THREAD

**Discussion by clankill3r** [helix-editor/helix#4180] [Q&A] [answered]
Hi,

Are there plans for supporting a debugger down the road?
I think that would be really great.

**Comment by Omnikar** [comment, marked as answer]
Debug adapter protocol is already (experimentally) supported, accessible via `<space>d`.

**Reply by helmesjo** [reply]
Nowadays it's `<space>g`.

**Comment by EricHenry** [comment]
Are there any conversations taken place on the future work for the Debugging feature?

**Reply by the-mikedavis** [reply]
There are some TODOs on https://github.com/helix-editor/helix/issues/505 and #574
